packages/teams-js/test/public/media.spec.ts

```diff
@@ -1215,6 +1333,47 @@ describe('media', () => {
         expect(blobContents).toEqual(stringMediaData);
       }
 
+      async function inValidateGetMediaMessageAndResults(
```

Comment:
New function to handle the error cases for the v2 tests.

I could have made this into one function, but I couldn't find a clean way to pass whether or not the tests were supposed to test for a success case or a rejected case, I would also have had to pass in the mediaResult array but this is straightforward.

If anyone has a suggestion, I would love to hear it